### PR TITLE
[IMP] account_avatax: find and apply exemptions for the shipping Country+State

### PR DESCRIPTION
**Pull request by dreispt**

### CODE REVIEW

**Review by atchuthan — APPROVED** [review]

**Comment by dreispt** [comment]
Thank you @atchuthan. I need to fix Travis before this can be merged.

File: account_avatax/models/account_move.py [hunk 1]
```diff
@@ -13,10 +13,45 @@ class AccountMove(models.Model):
...
+        - ACME Invoicing address, USA CA, no exemption status
+        - ACME Delivery address, USA CA, no exemption status
+
+        Invoice to ACME Invoicing, Shipped to ACME Delivery with be exempt.
+
+        For this to work properly, the "exemption_lock" is no longer supported.
+        """
+        for invoice in self:
+            invoice_partner = invoice.partner_id.commercial_partner_id
+            ship_to_address = invoice.shipping_add_id
```
Comment (atchuthan) [inline account_avatax/models/account_move.py]: There is no shipping_add_id in Odoo base code, it is partner_shipping_id

**Review by atchuthan — CHANGES_REQUESTED** [review]

File: account_avatax/models/account_move.py (hunk 1 above)
Comment (dreispt) [reply account_avatax/models/account_move.py]: Fixed, and build is green now

**Review by bodedra — APPROVED** [review]
Functional review. :+1: 

Tested with following two ways:

1. Manual invoice
2. Created SO and create invoice.

**Pull request merged**